### ISSUE TITLE
CPU governor does not switch back to powersave

### ISSUE THREAD

**Issue by hadrien-psydk** [FeralInteractive/gamemode#52]
I am on Ubuntu 18.04. In a terminal, I execute that command:
```
cat /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
powersave
powersave
...
```
Then I start Rise of the Tomb Raider. The window opens and I click on the Quit button at the bottom left.
Then I check the governor again in a terminal:
```
cat /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
performance
performance
...
```
I would expect gamemode to restore the CPU governor to powersave, as it was before starting Rise of the Tomb Raider.

Here is the output of ``systemctl --user status gamemoded.service``
```
hadrien@c18:~/code-tiers/gamemode$ systemctl --user status gamemoded.service 
● gamemoded.service - gamemoded
   Loaded: loaded (/etc/systemd/user/gamemoded.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2018-05-12 15:27:52 CEST; 10min ago
 Main PID: 17612 (gamemoded)
   Status: "GameMode is currently deactivated."
   CGroup: /user.slice/user-1000.slice/user@1000.service/gamemoded.service
           └─17612 /usr/bin/gamemoded -l

mai 12 15:38:14 c18 /usr/bin/gamemoded[17612]: Entering Game Mode...
mai 12 15:38:14 c18 /usr/bin/gamemoded[17612]: Requesting update of governor policy to performance
mai 12 15:38:14 c18 pkexec[18378]: pam_unix(polkit-1:session): session opened for user root by (uid=1000)
mai 12 15:38:14 c18 gamemoded[17612]: Setting governors to performance
mai 12 15:38:25 c18 /usr/bin/gamemoded[17612]: Removing expired game [18363]...
mai 12 15:38:25 c18 /usr/bin/gamemoded[17612]: Removing game: 18363 [/opt/steam_games/steamapps/common/Rise of the 
mai 12 15:38:25 c18 /usr/bin/gamemoded[17612]: Leaving Game Mode...
mai 12 15:38:25 c18 /usr/bin/gamemoded[17612]: Requesting update of governor policy to performance
mai 12 15:38:25 c18 pkexec[18454]: pam_unix(polkit-1:session): session opened for user root by (uid=1000)
mai 12 15:38:25 c18 gamemoded[17612]: Setting governors to performance
```

**Comment by hadrien-psydk** [comment]
It seems to work now: I ensured that my CPU was in powersave, I then did:
```
systemctl --user stop gamemoded
systemctl --user start gamemoded
```
If I understand correctly, gamemoded assumes the default governor mode is the one set when it starts, and not the one before the game is started?

**Comment by mdiluz** [comment]
Yeah, that's what it does at the moment.

It'd probably be fairly trivial to allow a config option to change that behaviour (and even set the default manually).

Just so I understand the situation more, what was causing your machine to be in performance when game mode started?

**Comment by hadrien-psydk** [comment]
Ok, thank you for the clarification. That behavior makes sense.

Indeed, I used to manually change the governor mode before playing and I forgot to restore it to powersave. I then installed gamemoded, hence my confusion. I went looking for a non-existing configuration file somewhere before I had the idea to restart the daemon. Maybe just adding a sentence in the readme would help?

Anyway, thank you for that software, it makes things more comfortable now. I think we can close this ticket.
